src/commands/shortcut.js

```diff
@@ -1,34 +1,29 @@
-import _ from 'lodash';
-import inquirer from 'inquirer';
-import {
-  loadExtensionJsonAsync,
-  saveExtensionJsonAsync,
-} from '../extension/data';
+import * as shortcut from '../extension/shortcut';
+import { loadExtensionJsonAsync, saveExtensionJsonAsync } from '../extension/data';
 import msg from '../user_messages';
 
-export async function createShortcut(shortcutName, screenName = null) {
+export async function createShortcut(shortcutName) {
```

Comment:
This comment applies to line 6. It would be great if we could have some kind of 'state manager'. It would contain info about authorization, current extension json file etc. That way you would not need to load ext.json all over the code, but just pull it from state manager